common/envs/theoremone.ts

```diff
@@ -12,6 +12,11 @@ export const THEOREMONE_CONFIG: EnvConfig = {
     appId: '1:698012149198:web:b342af75662831aa84b79f',
     measurementId: 'G-Y3EZ1WNT6E',
   },
+  // TODO: fill in real endpoints for T1
+  functionEndpoints: {
+    placebet: 'https://placebet-w3txbmd3ba-uc.a.run.app',
+    createmarket: 'https://createmarket-w3txbmd3ba-uc.a.run.app',
+  },
```

Comment:
I wonder if we should bother with keeping Theoremone up to date. It's a question for @akrolsmir if we can deprecate these.

Reply:
Austin earlier expressed to me that he was basically interested in spending zero effort to keep them up, so zero effort is what I am doing at the moment.